maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -155,12 +155,12 @@ public class MarkerState private constructor(position: LatLng) {
         /**
          * Creates a new [MarkerState] object
          *
-         * @param position the initial marker position
+         * @param initialPosition the initial marker position
          */
         @StateFactoryMarker
         public operator fun invoke(
-            position: LatLng = LatLng(0.0, 0.0)
-        ): MarkerState = MarkerState(position)
+            initialPosition: LatLng = LatLng(0.0, 0.0)
```

Comment:
I don't see the rationale for changing the parameter name here. Makes sense for rememberMarkerState, but not here. If you end up recomposing MarkerState.invoke(position) you will get a new MarkerState object.

Reply:
You are right. This part didn't need to be updated. 
However, I completely reverted and rewrote this part based on @kikoso 's opinion. 
Please review again. 🙇